applications/monitoring/values.yaml

```diff
@@ -77,20 +189,83 @@ cronjob:
     # -- taskmaker schedule
     taskmaker: "6-59/15 * * * *"
 
+# -- Gafaelfawr ingress for InfluxDBv2, only used if the service is enabled.
 ingress:
-  # -- ingress for Chronograf UI
-  chronograf:
-    # -- hostname for Chronograf UI
-    # @ default -- None, must be set by each individual instance
-    hostname: ""
+  enabled: false
+  influxdb2:
     # -- Additional annotations to add to the ingress
     annotations: {}
 
+# Telegraf for Prometheus monitoring.
+telegraf:
+  # -- enable telegraf at all?
+  enabled: true
+
+  # -- config for telegraf
+  # Use generated config instead of specifying anything here.
+  config:
+    processors: []
+    inputs: []
+    outputs: []
+
+  # -- pod resources for telegraf
```

Comment:
Needs a `@default` comment to avoid dumping JSON into the README.

Reply:
telegraf/-ds removed